tests/integration/models/reddit/test_wikipage.py

```diff
@@ -133,6 +140,17 @@ def test_add(self):
         with self.use_cassette():
             page.mod.add("bboe")
 
+    @mock.patch("time.sleep", return_value=None)
+    def test_hide_and_unhide(self, _):
+        subreddit = self.reddit.subreddit(pytest.placeholders.test_subreddit)
+        page = WikiPage(self.reddit, subreddit, "test")
+
+        self.reddit.read_only = False
+        with self.use_cassette():
+            revision_id = next(iter(page.revisions(limit=1)))["id"]
```

Comment:
Is `iter` necessary here?

Reply:
No, it's not necessary. I'll take it out.